### ISSUE TITLE
Installing from Google Fails --- Google Code is shut down

### ISSUE THREAD

**Issue by mikeholler** [sandstorm-io/capnproto#388]
I'm following the "From git" installation advice shown below. 

![image](https://cloud.githubusercontent.com/assets/1164112/20499395/efdae0a2-aff6-11e6-9301-76b2535b742a.png)

When I run `./setup-autotools.sh` I get:

```
================================================================================
Fetching Google Test code...
================================================================================
svn: E170013: Unable to connect to a repository at URL 'http://googletest.googlecode.com/svn/tags/release-1.7.0'
svn: E160013: '/svn/tags/release-1.7.0' path not found
```

This is because Google Code has been shut down. This repository was migrated to GitHub, and is now accessible via git at `https://github.com/google/googletest`. Can the `setup-autotools.sh` script be updated to pull from GitHub using git, to both mitigate this error and remove the subversion build requirement?

**Comment by dwrensha** [comment]
What version are you trying to build from git? `HEAD` of the master branch no longer requires autotools.

Apparently [capnproto.org](https://capnproto.org/install.html#installation-unix) has the outdated text that you quoted. That's surprising, especially because  [this commit](https://github.com/sandstorm-io/capnproto/commit/3ea03649781bca8dcc78b420cc5b6f16e4926aef) removed it. I'm guessing that @kentonv needs to clear some cache somewhere.

**Comment by dwrensha** [comment]
I meant "gtest" not autotools.

**Comment by mikeholler** [comment]
I was trying to build the latest git tag, `v0.5.3`. Is there really no official release that does not rely on google code?

**Comment by mikeholler** [comment]
Or should I be using the `release-0.5.3` branch instead?

**Comment by dwrensha** [comment]
OK, I've submitted https://github.com/sandstorm-io/capnproto/pull/389, which @kentonv can decide what to do with.

In the mean time, installing from released source tarballs should still work.

**Comment by mikeholler** [comment]
Thanks, I appreciate your help with this. The tarball should work for now.

**Comment by dwrensha** [comment]
Fixed by #389.
